### PR TITLE
[BACKEND-709] unleash downversion to php 5.6

### PR DESCRIPTION
**Pull request by kaisershahid**
## Goal

current state successfully runs through tests in php 5.6

## Reference

https://getpocket.atlassian.net/browse/BACKEND-709

### CODE REVIEW

File: tests/UnleashTest.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
 use GuzzleHttp\Handler\MockHandler;
 use GuzzleHttp\HandlerStack;
 use GuzzleHttp\Psr7\Response;
-use PHPUnit\Framework\Assert;
+use PHPUnit_Framework_Assert as Assert;
 use PHPUnit\Framework\TestCase;
```
Comment (bassrock) [inline tests/UnleashTest.php:10]: Does this need to change?

**Review by bassrock — APPROVED** [review]
👍

**Review by kkyeboah — APPROVED** [review]
lgtm

File: tests/UnleashTest.php (hunk 1 above)
Comment (kaisershahid) [reply tests/UnleashTest.php:10]: surprisingly no -- not sure why some classes use proper namespacing...

**Pull request merged**